blueprints/helper/files/app/helpers/__name__.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 
-export function <%= camelizedModuleName %>(input) {
-  return input;
+export function <%= camelizedModuleName %>(params, hash, options, env) {
```

Comment:
Helpers in core are not using `Ember.HTMLBars.makeBoundHelper`, they are not bound helpers.

Documentation is here:

https://github.com/emberjs/ember.js/blob/master/packages/ember-htmlbars/lib/system/make_bound_helper.js#L18-L60